rotkehlchen/tests/unit/decoders/test_makerdao_sai.py

```diff
@@ -2299,31 +2300,21 @@ def test_makerdao_sai_proxy_interaction(ethereum_transaction_decoder):
 @pytest.mark.vcr(filter_query_parameters=['apikey'])
 @pytest.mark.parametrize('ethereum_accounts', [['0xca482bCd75A6E0697aD6A1732aa187310b8372Df']])
```

Comment:
Hello @LefterisJP , the account checked in this test case was already a DSProxy address, I didn't change it. https://etherscan.io/address/0xca482bCd75A6E0697aD6A1732aa187310b8372Df 

So it seems that rotki already needed to check the DSProxy address for decoding some event.